### PR TITLE
Add priority parameter for website

### PR DESCRIPTION
**Pull request by andreaposadino**
Can be an idea to add a priority parameter for website to handle websites order

WIP

### CODE REVIEW

**Comment by mbiagetti** [comment]
after squash the fixup! remember to reword the commit 1dd41bf (`Add PriorityValueAlreadyUsedException` instead of `Add PriorityValueAlreadyUsedExceptionException`)

File: composer.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "description": "Symfony2 bundle to manage whitelabel",
   "license": "Apache-2.0",
   "type": "symfony-bundle",
+  "version": "1.1.0",
```
Comment (EmanueleMinotto) [inline composer.json]: https://getcomposer.org/doc/04-schema.md#version
> Optional if the package repository can infer the version from somewhere, such as the VCS tag name in the VCS repository. In that case it is also recommended to omit it.

**Comment by mbiagetti** [comment]
After merge create a new tags/release, please

File: DependencyInjection/AeWhiteLabelExtension.php [hunk 2]
```diff
@@ -61,4 +69,44 @@ protected function configureWhiteLabel($name, array $website, ContainerBuilder $
...
+    {
+        if (key_exists($priority, $priorities)) {
+            throw new PriorityValueAlreadyUsedException(
+                sprintf(
+                    'priority %s for website %s is already used by %s',
+                    $priority,
+                    $name,
+                    $priorities[$priority]
+                )
+            );
```
Comment (daBarbe) [inline DependencyInjection/AeWhiteLabelExtension.php]: You could modify the constructor so you don't have to specify the message here.

File: Tests/DependencyInjection/AeWhiteLabelExtensionTest.php [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+            'hootsuite' => 'ae_white_label.website.hootsuite',
+            'adespresso' => 'ae_white_label.website.adespresso',
+        ];
+
+        $this->assertSame(
+            $this->container->getParameter('ae_white_label.websites'),
+            $expectedWebsites,
+            'Check sorting by the whitelabel priority'
+        );
+    }
```
Comment (daBarbe) [inline Tests/DependencyInjection/AeWhiteLabelExtensionTest.php:171]: Change this test to remove the real names (but do not delete it).

**Review by daBarbe — CHANGES_REQUESTED** [review]
Just a couple of changes for you.

**Review by daBarbe — APPROVED** [review]

**Comment by daBarbe** [comment]
LGTM :+1:

Squash the fixups, rebase, and 🚢  !

#approved

**Pull request merged**